app/1.2.x/admin-api.md

```diff
@@ -0,0 +1,2731 @@
+```
+
+- **application/json**
+
+Handy for complex bodies (ex: complex plugin configuration), in that case simply send
+a JSON representation of the data you want to send. Example:
+
+```json
+{
+    "config": {
+        "limit": 10,
+        "period": "seconds"
+    }
+}
+```
+
+---
+
+## Information Routes
+
+
+
+### Retrieve Node Information
+
+Retrieve generic details about a node.
+
+<div class="endpoint get">/</div>
+
+*Response*
+
+```
+HTTP 200 OK
+```
+
+```json
+{
+    "hostname": "",
+    "node_id": "6a72192c-a3a1-4c8d-95c6-efabae9fb969",
+    "lua_version": "LuaJIT 2.1.0-beta3",
+    "plugins": {
+        "available_on_server": [
+            ...
+        ],
+        "enabled_in_cluster": [
+            ...
+        ]
+    },
+    "configuration" : {
+        ...
+    },
+    "tagline": "Welcome to Kong",
+    "version": "0.14.0"
+}
+```
+
+* `node_id`: A UUID representing the running Kong node. This UUID
+  is randomly generated when Kong starts, so the node will have a
+  different `node_id` each time it is restarted.
+* `available_on_server`: Names of plugins that are installed on the node.
+* `enabled_in_cluster`: Names of plugins that are enabled/configured.
+  That is, the plugins configurations currently in the datastore shared
+  by all Kong nodes.
+
+
+---
+
+### Retrieve Node Status
+
+Retrieve usage information about a node, with some basic information
+about the connections being processed by the underlying nginx process,
+the status of the database connection, and node's memory usage.
+
+If you want to monitor the Kong process, since Kong is built on top
+of nginx, every existing nginx monitoring tool or agent can be used.
+
+
+<div class="endpoint get">/status</div>
+
+*Response*
+
+```
+HTTP 200 OK
+```
+
+```json
+{
+    "memory": {
+		"workers_lua_vms": [{
+			"http_allocated_gc": "0.02 MiB",
+			"pid": 18477
+		}, {
+			"http_allocated_gc": "0.02 MiB",
+			"pid": 18478
+		}],
+		"lua_shared_dicts": {
+			"kong": {
+				"allocated_slabs": "0.04 MiB",
+				"capacity": "5.00 MiB"
+			},
+			"kong_db_cache": {
+				"allocated_slabs": "0.80 MiB",
+				"capacity": "128.00 MiB"
+			},
+		}
+	},
+    "server": {
+        "total_requests": 3,
+        "connections_active": 1,
+        "connections_accepted": 1,
+        "connections_handled": 1,
+        "connections_reading": 0,
+        "connections_writing": 1,
+        "connections_waiting": 0
+    },
+    "database": {
+        "reachable": true
+    }
+}
+```
+
+* `memory`: Metrics about the memory usage.
+    * `workers_lua_vms`: An array with all workers of the Kong node, where each
+      entry contains worker memory usage information (`http_allocated_gc`) and
+      its process identification number (`pid`).
+    * `lua_shared_dicts`: An array of information about shared dictionaries,
+      where each node contains how much memory is dedicated for the specific
+      shared dictionary (`capacity`) and how much of said memory is in use
+      (`allocated_slabs`).
```

Comment:
Here, we should clarify that shared dictionaries have LRU eviction capabilities, meaning that a `lua_shared_dict` with `allocated_slabs == capacity` will still be able to function properly. That said, a note can be made that for _some_ shms (e.g. cache hit/miss shms), increasing their size if they are full can be beneficiary to the overall performance of a Kong node.

Reply:
We should also mention that lua_shared_dicts are mapped memory zones shared by all Nginx workers.